docs/developers_guide/ocean/framework.md

```diff
@@ -85,6 +136,168 @@ The function {py:func}`polaris.ocean.mesh.spherical.add_spherical_base_mesh_step
 returns a step for for a spherical `qu` or `icos` mesh of a given resolution 
 (in km).  The step can be shared between tasks.
 
+(dev-ocean-spherical-convergence)=
+
+## Spherical Convergence Tests
+
+Several tests that are in Polaris or which we plan to add are convergence
+tests on {ref}`dev-ocean-spherical-meshes`. The ocean framework includes
+shared config options and a base class for forward steps that are expected
+to be useful across these tests.
+
+The shared config options are:
+```cfg
+# config options for spherical convergence tests
+[spherical_convergence]
+
+# a list of icosahedral mesh resolutions (km) to test
+icos_resolutions = 60, 120, 240, 480
+
+# a list of quasi-uniform mesh resolutions (km) to test
+qu_resolutions = 60, 90, 120, 150, 180, 210, 240
+
+# time integrator: {'split_explicit', 'RK4'}
+time_integrator = RK4
+
+# RK4 time step per resolution (s/km), since dt is proportional to resolution
+rk4_dt_per_km = 3.0
+
+# split time step per resolution (s/km), since dt is proportional to resolution
+split_dt_per_km = 30.0
+
+# the barotropic time step (s/km) for simulations using split time stepping,
+# since btr_dt is proportional to resolution
+btr_dt_per_km = 1.5
+
+# Run duration in days
+run_duration = 1.0
```

Comment:
@xylar Can you say a bit more about your decision to include this in the `spherical_convergence` section? It seems like the kind of thing that's not likely to be shared across test cases.

Reply:
I'm definitely open to changing this.  

I don't think the config options here are meant to be common ones that convergence tests are unlikely to change.  Rather, they are meant to be indications of what the config options are that a convergence test *can* (and maybe should) change when using this shared functionality.  It seemed from past experience that showing more simply what config options are used in the shared infrastructure was better than having it hidden in the code.  If they don't appear in `spherical.cfg` but they are required in the code, it's a little harder for a developer to anticipate that (even if it's documented).

Currently, I don't have a config section getting passed in to `Forward`, so all the config options have to be in the `spherical_convergence` section.  But that could be changed.

Reply:
We can also leave my suggestion to the second PR with the new analysis step.

Reply:
I'm fine with this being the convergence time, but it really is the run duration that is needed in `Forward` so there still would be a need to get that config option somehow.  How would you suggest doing that?

I'd rather get this fixed the way you like here.  Adding more pieces can wait but I'd like to have the `Forward` part and the overall approach (e.g. to config options) to be decided on here.

Reply:
Regarding `run_duration`, is there a different but generic config section that should go in instead?

Reply:
I meant that we could keep `run_duration` in this section, equal to `${convergence_eval_time}` (or whatever we decide to call it). I don't think we have a great alternative to test case-specific sections right now.

Reply:
I see. That works for me.

Reply:
Thanks!

Reply:
This has been added.  I also broke the config options into `[spherical_convergence]` and `[spherical_convergence_forward]`, in part in anticipation of the work I'm doing on config options for shared steps and in part for clarity.